unicodetools/data/ucd/dev/DerivedCoreProperties.txt

```diff
@@ -7438,6 +7438,7 @@ FFDA..FFDC    ; ID_Start # Lo   [3] HALFWIDTH HANGUL LETTER EU..HALFWIDTH HANGUL
 1FE0..1FEC    ; ID_Continue # L&  [13] GREEK SMALL LETTER UPSILON WITH VRACHY..GREEK CAPITAL LETTER RHO WITH DASIA
 1FF2..1FF4    ; ID_Continue # L&   [3] GREEK SMALL LETTER OMEGA WITH VARIA AND YPOGEGRAMMENI..GREEK SMALL LETTER OMEGA WITH OXIA AND YPOGEGRAMMENI
 1FF6..1FFC    ; ID_Continue # L&   [7] GREEK SMALL LETTER OMEGA WITH PERISPOMENI..GREEK CAPITAL LETTER OMEGA WITH PROSGEGRAMMENI
+200C..200D    ; ID_Continue # Cf   [2] ZERO WIDTH NON-JOINER..ZERO WIDTH JOINER
```

Comment:
As Mark said, this should be generated, using the Other_ID_... properties as input. Did you edit these manually?

I can try to make equivalent changes and turn the crank.

Reply:
> Did you edit these manually?

Yes, since I have not been able to get things to work on Windows. I was expecting some CI test to check that it was the same as the output of the generator.

Reply:
(At least in the case of #388 we had a bunch of tests that caught generated files that I forgot to update.)

Reply:
I cloned your PR branch, ran the tools, applied the timestamp change, pushed, and will squash-and-merge when the CI checks are happy. I can report that you managed to manually change the derived data exactly as the tool does :-)